### PR TITLE
Make the ff3v fir coef arrays usable from C and fix alignment

### PR DESCRIPTION
**Pull request by mbruno-xmos**
This makes the fixed factor of 3 voice functions usable from C.

### CODE REVIEW

**Comment by mbruno-xmos** [comment]
It no longer is usable from XC. Can somebody who understands XC please fix this so that the voice up/downsample functions are callable from both XC and C? The fact that a global array needs to be passed to these (the fir array) seems to be a real problem for making it usable from both C and XC.

**Comment by mbruno-xmos** [comment]
I came up with a way to make the src_ff3v_fir_coefs array usable from both C and XC. Seems a little crazy. Is there a better way to do this?

File: lib_src/src/fixed_factor_of_3_voice/src_ff3v_fir.h [hunk 1]
```diff
@@ -21,6 +21,13 @@ extern const unsigned src_ff3v_fir_comp_q_us;
 extern const int32_t src_ff3v_fir_comp_us;
 
 extern int32_t src_ff3v_fir_coefs_debug[SRC_FF3V_FIR_NUM_PHASES * SRC_FF3V_FIR_TAPS_PER_PHASE];
```
Comment (mbanth) [inline lib_src/src/fixed_factor_of_3_voice/src_ff3v_fir.h:23]: The non-const debugging array has not been made usable from C.  Is that your intention?

**Review by mbanth — CHANGES_REQUESTED** [review]
The changes look good, @mbruno-xmos.  I have only one question and two requests.  Please update the version to 2.2.0 in the module_build_info file and add the version header and an entry describing the change in the CHANGELOG.rst file.

File: lib_src/src/fixed_factor_of_3_voice/src_ff3v_fir.h (hunk 1 above)
Comment (mbruno-xmos) [reply lib_src/src/fixed_factor_of_3_voice/src_ff3v_fir.h:23]: It is already usable from C. It seems that single dimensional arrays don't throw the same linker error that multi-dimensional arrays do when sharing them between C and XC.

**Review by mbanth — APPROVED** [review]
Looks good.

**Pull request merged**